packages/svelte/src/reactivity/map.js

```diff
@@ -38,6 +40,9 @@ export class ReactiveMap extends Map {
 	#increment_version() {
 		set(this.#version, this.#version.v + 1);
 	}
+	#increment_inspect_version() {
+		set(this.#inspect_version, this.#inspect_version.v + 1);
```

Comment:
This doesn't feel right. This will be the "cause" for the mutation, when in reality it should be the actual set?

Reply:
you got a point, let me remove this and do a get on all the sources instead.